src/main/resources/jenkins/plugins/openstack/compute/SlaveOptions/help-metadata.html

```diff
@@ -0,0 +1,3 @@
+<div>
+  Additional metadata options using "name1=value1,name1=value2" format.
```

Comment:
I suggest to make it multi-line instead of `,` delimited. Also, consider using `Properties` parsing not to surprise Jenkins users that are used to the syntax.